### PR TITLE
ig: Bump version to v0.26.0.

### PR DESCRIPTION
**Pull request by eiffel-fl**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Bump `ig` version to `v0.26.0`.

###### Change Log  

- Bump to version 0.26.0.

###### Does this affect the toolchain?  

**NO**

### CODE REVIEW

**Comment by mfrw** [comment]
Closing & Re-Opening to retrigger Stuck Github PR Checks.

**Comment by christopherco** [comment]
Requeued https://dev.azure.com/mariner-org/mariner/_build/results?buildId=541414&view=results

File: SPECS/ig/ig.signatures.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "ig-0.25.0-govendor-v1.tar.gz": "ba1c9153ee67cd9d1ebd31021ca3848c1099979c229b26bea6ad05470ac486ac",
-  "ig-0.25.0.tar.gz": "3d283e0f6e758b36e291e483e79e7a08c4ce2965a6c11f5c1ab671ced7015a6e"
+  "ig-0.26.0-govendor-v1.tar.gz": "260c4d8f9864cc3dd54567a854c5ab8dcb0c8246cf1caa1d3cb54c8e25328b90",
```
Comment (christopherco) [inline SPECS/ig/ig.signatures.json:3]:
```suggestion
  "ig-0.26.0-govendor-v1.tar.gz": "79bd7c40cb29f4eb7e8f002f9ad8bc30928c48bea2ea6766e73d18c36513b775",
```
This was the hash when I generated the vendored go. Checking if the difference is expected.

```bash
bash generate_source_tarball.sh --srcTarball ig-0.26.0.tar.gz --pkgVersion 0.26.0
```

Comment (mfrw) [reply SPECS/ig/ig.signatures.json:3]: I think I should create one locally to see if there is a correlation with respect to environment.

Comment (mfrw) [reply SPECS/ig/ig.signatures.json:3]:
```bash
mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$?] 
fsh ❯ bash generate_source_tarball.sh --srcTarball ig-0.26.0.tar.gz --pkgVersion 0.26.0
--srcTarball      -> ig-0.26.0.tar.gz
--outFolder       -> /home/mfrw/mariner-org/3.0/SPECS/ig
--pkgVersion      -> 0.26.0
--vendorVersion   -> 1
-- create temp folder
Unpacking source tarball...
Vendor go modules...

=========================
Tar vendored tarball
ig vendored modules are available at /home/mfrw/mariner-org/3.0/SPECS/ig/ig-0.26.0-govendor-v1.tar.gz
+++ cleanup -> remove /tmp/tmp.mo0Q0In2KL
⏎                                                                                                         

mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ ls
generate_source_tarball.sh  ig-0.26.0-govendor-v1.tar.gz  ig-0.26.0.tar.gz  ig.signatures.json  ig.spec
mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ cat ig.signatures.json 
{
 "Signatures": {
  "ig-0.26.0-govendor-v1.tar.gz": "260c4d8f9864cc3dd54567a854c5ab8dcb0c8246cf1caa1d3cb54c8e25328b90",
  "ig-0.26.0.tar.gz": "22268a32daccacfc01afdb66942c75ba0394d47c9df4f19720968dd6a8ad758d"
 }
}⏎

mfrw in 🌐 curie in ~/ma/3.0/SPECS/ig on  francis/bump-ig:git@github.com:eiffel-fl/CBL-Mariner.git/francis/bump-ig [$!?] 
fsh ❯ go version
go version go1.22.3 linux/amd64
```

Comment (mfrw) [reply SPECS/ig/ig.signatures.json:3]: @christopherco - I will share the vendor tar ball to you.
As we discussed offline, it was indeed go `1.21` vs `1.22` :)

**Comment by christopherco** [comment]
Rebuilt and tests look good - https://dev.azure.com/mariner-org/mariner/_build/results?buildId=577397&view=results

**Review by christopherco — APPROVED** [review]

**Pull request merged**